html/components/dropdown.js

```diff
@@ -4,12 +4,12 @@ import { isEscPressed } from '../utilities/keypress';
 
 const hideDropDown = (dropdown) => {
   dropdown.classList.remove('sprk-c-Dropdown--open');
-  dropdown.classList.add('sprk-u-Display--none');
+  dropdown.classList.add('sprk-c-Dropdown--is-hidden');
 };
 
 const showDropDown = (dropdown) => {
   dropdown.classList.add('sprk-c-Dropdown--open');
```

Comment:
Just a note on this one, I did not see dropdown-open in the styles and it looks like it is being used as a js hook to determine whether the dropdown is open or not based on the showDropdown function. THis is fine for now, but we could increase the performance of the component by removing this hook addition and deletion in the future. We'll get an issue in.